packages/server/src/opencrvs-api.ts

```diff
@@ -126,3 +126,76 @@ export const upsertRegistrationIdentifier = (
     },
     headers,
   });
+
+export const getOIDPUserInfo = (
```

Comment:
There might have been some confusion here, when I said: "replicate the graphQL Query "getOIDPUserInfo" as a REST endpoint".  I meant that the functionality that used to be a GraphQL mutation and resolver in gateway, should be moved into esignet-api.ts